staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -77,6 +77,10 @@ type Interface interface {
 
 	// WatchTracker provides the WatchTracker interface.
 	WatchTracker
+
+	// PriorityLevelMaxSeats returns the maximum seats used by work estimator
+	// for a given priority level
+	PriorityLevelMaxSeats(plName string) uint64
```

Comment:
nit: I would prefere defining a type (exporting `maxSeatsFunc` type?) and using
```
PriorityLevelMaxSeats MaxSeatsFunc
```

Reply:
ack -- this is in an interface, so it would have to be:
```
PriorityLevelMaxSeats() MaxSeatsFunc
```

Reply:
done

Reply:
Yeah, I do not understand @wojtek-t's comment either, not sure a change was needed here.